doc/Learn/Tutorials/Maui/HowTo-Use3rdPartLibs.md

```diff
@@ -2,17 +2,17 @@
 uid: Learn.Tutorials.Maui.HowToUse3rdPartLibs
 ---
 
-# How-To: Install and use third part libraries on Uno apps with Maui Embedding
+# How-To: Install and use third-part libraries on Uno apps with Maui Embedding
 
-When working in an application project, you may need to install a third party library. This guide will show you how to install and use a third party library in your Uno app with Maui Embedding.
+When working on an application project, you may need to install a third-party library. This guide will show you how to install and use a third-party library in your Uno app with Maui Embedding.
 
-For this sample we will use the `CommunityToolkit.Maui` library.
+For this sample, we will use the `CommunityToolkit.Maui` library.
 
 ## Step-by-steps
 
 ### 1. Installing the NuGet package
 
-On Visual Studio you can use the `Nuget Package Manager` to install the `CommunityToolkit.Maui` package. Install it on your class library project and mobile and Windows project (because this package support those platforms).
+On Visual Studio, you can use the `Nuget Package Manager` to install the `CommunityToolkit.Maui` package. Install it on your class library project and mobile and Windows project (because this package support those platforms).
```

Comment:
```suggestion
On Visual Studio, you can use the `Nuget Package Manager` to install the `CommunityToolkit.Maui` package. Install it on your class library project and mobile and Windows projects (because this package supports those platforms).
```

Reply:
Fixed in my next commit.